### ISSUE TITLE
Reference state of humid air (HAPropsSI)

### ISSUE THREAD

**Issue by LorenzSLH** [CoolProp/CoolProp#2045]
### Description
I am very well aware that the enthalpy and entropy are both relative dimensions and therefore have a reference point. Is there a possibility to read out the reference state used in the humid air model or even change it? 

Or is it using the reference state of air and can be changed with the coolprop.set_reference_state command for air? 

with best regards 
Lorenz 

**CoolProp Version:**  6.4.1
**Operating System and Version:** Windows 10
**Access Method:** Python

**Comment by ibell** [comment]
You can read the report the psychrometric properties are based on.  I don't recall off the top of my head; I remember it was confusing, and poorly documented.

**Comment by dododododo21** [comment]
Hi 
I checked the values of the saturated enthalpy and entropy with the ASHRAE_21 tables. They vary at max 0.05%.
Then I tried to change the reference state from water and air. The enthalpies with the PropsSi() function changed for air and water but not with HAPropsSI(). 
Later I saw this in the HumidAirProps.cpp code in MolarEnthalpy line1021: "If the user changes the reference state for water or Air, we need to ensure that the values returned from this function are always the same as the formulation expects.  Therefore we can use a state point for which we know what the enthalpy should be and then correct the calculated values for the enthalpy."  
So i think it is impossible to change it for humid air?
In line 1033 you set hbar_0 equal zero and question yourself, why it doesn't need this coefficient equal 2.924425468.
I think this is because you are using a different reference point than the RP1485.
In the function IdealGasMolarEnthalpy_Water you are using T=473.15K,...
While the RP1485 uses the triple point and a different enthalpy
From where do you have this Reference Point?

All the other properties of moist air were identical to the values from ASHRAE. Great Work, Thank you
Greetings Dominik

**Comment by ibell** [comment]
@dododododo21 You are correct, the reference state for humid air cannot be changed.  The way the reference states are described in RP-1485 is quite unclear. So I fudged the results to get the "right" answer.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
